### ISSUE TITLE
IndexError thrown every time ST starts

### ISSUE THREAD

**Issue by rwols** [jisaacks/GitGutter#528]
## Description

Every time I open sublime text, I see this exception in the console:

```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 462, in run_callback
    expr()
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 586, in <lambda>
    run_callback('on_selection_modified', callback, lambda: callback.on_selection_modified(v))
  File "/Users/raoulwols/Library/Application Support/Sublime Text 3/Installed Packages/GitGutter.sublime-package/modules/events.py", line 278, in on_selection_modified
    self._run_blame(view, False)
  File "/Users/raoulwols/Library/Application Support/Sublime Text 3/Installed Packages/GitGutter.sublime-package/modules/events.py", line 289, in _run_blame
    row, _ = view.rowcol(selection[0].begin())
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime.py", line 649, in __getitem__
    raise IndexError()
IndexError
```

## Support Info

- Sublime Text 3180
- Platform: osx
- Arch: x64
- GitGutter 1.9.4
- Install via PC: True
- git version 2.17.2 (Apple Git-113)
- mdpopups 3.3.3
- markdown 2.6.11
- pygments 2.1a0
- jinja2 2.8

## Steps to Reproduce Issue

1. Close ST.
2. Open ST.
3. Open console.

**Comment by deathaxe** [comment]
Can you checkout #529 please and try whether it fixes your issue and does not introduce any new side effects?

**Comment by deathaxe** [comment]
Fixed in 1.9.5
